ecl/resource_ecl_mlb_policy_v1.go

```diff
@@ -153,41 +202,85 @@ func resourceMLBPolicyV1Read(d *schema.ResourceData, meta interface{}) error {
 	} else if policy.ConfigurationStatus == "UPDATE_STAGED" {
 		d.Set("algorithm", ternary(policy.Staged.Algorithm == "", policy.Algorithm, policy.Staged.Algorithm))
 		d.Set("persistence", ternary(policy.Staged.Persistence == "", policy.Persistence, policy.Staged.Persistence))
+		d.Set("persistence_timeout", ternary(policy.Staged.PersistenceTimeout == 0, policy.PersistenceTimeout, policy.Staged.PersistenceTimeout))
 		d.Set("idle_timeout", ternary(policy.Staged.IdleTimeout == 0, policy.IdleTimeout, policy.Staged.IdleTimeout))
 		d.Set("sorry_page_url", ternary(policy.Staged.SorryPageUrl == "", policy.SorryPageUrl, policy.Staged.SorryPageUrl))
 		d.Set("source_nat", ternary(policy.Staged.SourceNat == "", policy.SourceNat, policy.Staged.SourceNat))
 		d.Set("certificate_id", ternary(policy.Staged.CertificateID == "", policy.CertificateID, policy.Staged.CertificateID))
 		d.Set("health_monitor_id", ternary(policy.Staged.HealthMonitorID == "", policy.HealthMonitorID, policy.Staged.HealthMonitorID))
 		d.Set("listener_id", ternary(policy.Staged.ListenerID == "", policy.ListenerID, policy.Staged.ListenerID))
 		d.Set("default_target_group_id", ternary(policy.Staged.DefaultTargetGroupID == "", policy.DefaultTargetGroupID, policy.Staged.DefaultTargetGroupID))
+		d.Set("backup_target_group_id", ternary(policy.Staged.BackupTargetGroupID == "", policy.BackupTargetGroupID, policy.Staged.BackupTargetGroupID))
 		d.Set("tls_policy_id", ternary(policy.Staged.TLSPolicyID == "", policy.TLSPolicyID, policy.Staged.TLSPolicyID))
 	} else if policy.ConfigurationStatus == "DELETE_STAGED" {
 		d.SetId("")
 		return nil
 	}
 
+	if policy.ConfigurationStatus == "ACTIVE" || (policy.ConfigurationStatus == "UPDATE_STAGED" && policy.Staged.ServerNameIndications == nil) {
+		serverNameIndications := make([]map[string]interface{}, len(policy.ServerNameIndications))
+		for i, serverNameIndication := range policy.ServerNameIndications {
+			serverNameIndications[i] = map[string]interface{}{
+				"server_name":    serverNameIndication.ServerName,
+				"input_type":     serverNameIndication.InputType,
+				"priority":       serverNameIndication.Priority,
+				"certificate_id": serverNameIndication.CertificateID,
+			}
+		}
+		d.Set("server_name_indications", serverNameIndications)
+	} else if (policy.ConfigurationStatus == "CREATE_STAGED" || policy.ConfigurationStatus == "UPDATE_STAGED") && policy.Staged.ServerNameIndications != nil {
+		serverNameIndications := make([]map[string]interface{}, len(policy.Staged.ServerNameIndications))
+		for i, serverNameIndication := range policy.Staged.ServerNameIndications {
+			serverNameIndications[i] = map[string]interface{}{
+				"server_name":    serverNameIndication.ServerName,
+				"input_type":     serverNameIndication.InputType,
+				"priority":       serverNameIndication.Priority,
+				"certificate_id": serverNameIndication.CertificateID,
+			}
+		}
+		d.Set("server_name_indications", serverNameIndications)
```

Comment:
[NITS]
細かくて申し訳ないのですが、空行を入れて頂けると助かります

```suggestion

		d.Set("server_name_indications", serverNameIndications)
```

Reply:
空行を追加しました